funnel/views/session.py

```diff
@@ -203,7 +210,8 @@ def delete(self):
                 status=True,
                 modal_url=modal_url,
                 msg=_(
-                    "This project will not be listed as it has no sessions in the schedule"
+                    "This project will not be listed as it has no sessions in the"
+                    " schedule"
```

Comment:
Our convention for wrapped lines is to put the space at the end of the previous line.